src/generate.rs

```diff
@@ -24,28 +24,6 @@ pub trait Generate<Cfg> {
     fn generate(self, codemap::Span, &mut base::ExtCtxt, Cfg) -> P<ast::Item>;
 }
 
-fn block_to_stmts(block: &ast::Block) -> Vec<ast::Stmt> {
-    use syntax::codemap::Spanned;
-
-    let block = block.clone();
-
-    let id = block.id;
-    let stmts = block.stmts;
-    let expr = block.expr;
```

Comment:
This does not exist anymore. When I remove this, the whole code is now useless and so I simplified the whole file.

The change in Rust is https://github.com/rust-lang/rust/commit/9bb3ea0febcbb8f6d7715256a5644daa985cf4e7

Also I may be wrong in my analysis so please tell me if something is broken :)